Notebooks/verify_truth.ipynb

```diff
@@ -0,0 +1,724 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Light curves\n",
+    "Find a reasonable candidate (maybe one each agn, lensed agn, lensed sn?) and compare light curves.  Use just PostgreSQL initially to find the candidate because it can easily do joins."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def format_cone_search(coord_column, ra, dec, radius):\n",
+    "    '''\n",
+    "    Parameters\n",
+    "    coord_column:  name of column of type earth in the table\n",
+    "    ra:  ra value at center of cone (degrees)\n",
+    "    dec:  dec value at center of cone (degrees)\n",
+    "    radius: radius of cone (arcseconds)\n",
+    "    \n",
+    "    Returns\n",
+    "    Condition to be inserted into WHERE clause for the query\n",
+    "    '''\n",
+    "    cond = f\"\"\"conesearch({coord_column},'{ra}','{dec}','{radius}')\"\"\"\n",
+    "    return cond"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# pick a location that probably gets lots of visits\n",
+    "ra = 53.0      \n",
+    "dec = -28.1\n",
+    "radius = 80   \n",
+    "truth_schema = 'agn_truth'\n",
+    "tbl_spec = f\"\"\"SELECT S.id, S.ra, S.dec, max(abs(V.delta_flux)) as max_delta_flux,count(V.bandpass) AS visit_count \n",
+    "           FROM {truth_schema}.truth_summary AS S JOIN \n",
+    "           {truth_schema}.agn_variability_truth AS V ON S.id=V.id \"\"\"\n",
+    "where = \"WHERE \" + format_cone_search('S.coord', ra, dec, radius) + \" AND S.is_variable=1 \"\n",
+    "group_by = \" GROUP BY S.id,S.ra,S.dec ORDER BY max_delta_flux DESC\"\n",
+    "q = tbl_spec + where + group_by\n",
+    "print(q)\n",
+    "\n",
+    "# This takes a couple minutes to complete\n",
+    "dbconn = Catalog.pg_connection()\n",
+    "with dbconn.cursor() as cursor:\n",
+    "    %time cursor.execute(q)\n",
+    "    records = cursor.fetchall()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df_lengths = pd.DataFrame(records, columns=['id', 'ra','dec', 'max_delta_flux','visit_count'])\n",
+    "df_lengths"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot light curves for one star\n",
+    "Pick first object for large delta flux\n",
+    "#### Get the data\n",
+    "For each data source et delta_flux and time values for the plot and some summary information about the star. Order conveniently for plotting."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#id = df_lengths['id'][0]\n",
```

Comment:
This cell is completely commented out. Do you intend to leave it in there?

Reply:
No, I just forgot about it. It should go.